library/UNICODE

```diff
@@ -1194,8 +1191,8 @@
 )
 (DECLARE%: EVAL@COMPILE DONTCOPY 
 
-(OR (GETP 'EXPORTS.ALL 'FILE)
-    (PRINT "NOTE: UNICODE requires EXPORTS.ALL for compilation" T))
+(FILESLOAD (FROM LOADUPS)
+       EXPORTS.ALL)
```

Comment:
shouldn't this and others be
(FILES (FROM LOADUPS) EXPORTS.ALL)
Instead of (P (FILESLOAD ...) ...)